### ISSUE TITLE
Fix Package Deprecation Warnings

### ISSUE THREAD

**Issue by Morgul** [PieceMaker/max-inscribed-circle#16]
When installing, there are several package deprecation warnings:

```bash
[ccase:dt-ccase-lnx] ~/d/max-test > npm install --save-dev max-inscribed-circle
npm WARN deprecated turf-area@1.1.1: Turf packages are now namespaced: please use @turf/area instead
npm WARN deprecated turf-bearing@1.0.2: Turf packages are now namespaced: please use @turf/bearing instead
npm WARN deprecated turf-within@1.0.1: Turf packages are now namespaced: please use @turf/within instead
npm WARN deprecated turf-centroid@3.0.12: Turf packages are now namespaced: please use @turf/centroid instead
npm WARN deprecated turf-destination@1.2.1: Turf packages are now namespaced: please use @turf/destination instead
npm WARN deprecated turf-distance@1.1.0: Turf packages are now namespaced: please use @turf/distance instead
npm WARN deprecated geojson-area@0.2.1: This module is now under the @mapbox namespace: install @mapbox/geojson-area instead
npm WARN deprecated turf-helpers@3.0.12: Turf packages are now namespaced: please use @turf/helpers instead
npm WARN deprecated turf-meta@3.0.12: Turf packages are now namespaced: please use @turf/meta instead
npm WARN deprecated turf-invariant@1.0.3: Turf packages are now namespaced: please use @turf/invariant instead
npm WARN deprecated turf-inside@1.1.4: Turf packages are now namespaced: please use @turf/inside instead
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN max-test@1.0.0 No description
npm WARN max-test@1.0.0 No repository field.

+ max-inscribed-circle@1.0.5
```

**Comment by PieceMaker** [comment]
These warnings were fixed with #13. I will try to publish the latest changes to NPM this weekend.

**Comment by Morgul** [comment]
Cool, then we can close this.
